### PR TITLE
style: PEP8, simplified statements

### PR DESCRIPTION
**Pull request by ksandwojtek**
Everything should be good now, I accidentally closed the other PR cause im pretty new to github

### CODE REVIEW

**Review by auroraisluna — APPROVED** [review]
Looks good to me. 

There are some lines where you are not using the PEP8 format: 
```py
thing(var,
          var2)
```
instead you are using: 
```py
thing(
          var,
          var2
          )
```

I personally like the latter, but for consistency, please ensure everything matches PEP8 or the latter.

**Comment by ksandwojtek** [comment]
Should be fixed now

File: src/GuiThread.py [hunk 1]
```diff
@@ -35,11 +35,9 @@ def generateTable(self):
 
         for acc in self.stats.accountData:
             status = self.stats.accountData[acc]["status"]
-            table.add_row(
-                f"{acc}", f"{status}", f"{self.stats.accountData[acc]['liveMatches']}",
-                f"{self.stats.accountData[acc]['lastCheck']}", f"{self.stats.accountData[acc]['lastDrop']}",
-                f"{self.stats.accountData[acc]['totalDrops']}"
-            )
+            table.add_row(f"{acc}", f"{status}", f"{self.stats.accountData[acc]['liveMatches']}",
```
Comment (auroraisluna) [inline src/GuiThread.py:38]: why aren't these split in multiple lines?

**Review by auroraisluna — CHANGES_REQUESTED** [review]

File: src/GuiThread.py (hunk 1 above)
Comment (ksandwojtek) [reply src/GuiThread.py]: > why aren't these split in multiple lines?

fixed, its pretty late, I didn't see it

**Pull request closed without merging**